src/core/operations_problem.jl

```diff
@@ -365,6 +463,7 @@ function _build!(problem::OperationsProblem{<:AbstractOperationsProblem}, serial
             serialize && serialize_problem(problem)
             serialize && serialize_optimization_model(problem)
             set_status!(problem, BuildStatus.BUILT)
+            log_values(get_settings(problem))
```

Comment:
@jd-lara I added this so that we can have an easy-to-access record of what the user set for time_series_cache_size. I think it's a good addition overall, but let me know if you find it too noisy. It could be debug.